content/en/monitors/monitor_types/log.md

```diff
@@ -10,88 +10,68 @@ further_reading:
-
-5. Set alert conditions. The following options can be used:
-
-  {{< img src="monitors/monitor_types/log/above_below.png" alt="aggregation function for Log Analytics" responsive="true" style="width:50%;">}}
+A logs monitor alerts when a specified type of log exceeds a user-defined threshold over a given period of time.
 
-6. Configure your **notification options**:  
+## Monitor creation
 
-  {{< img src="monitors/monitor_types/log/set_alert_conditions.png" alt="Set alert conditions" responsive="true" style="width:50%;" >}}
+To create a [logs monitor][1] in Datadog, use the main navigation: *Monitors --> New Monitor --> Logs*.
 
-  Refer to the [notifications][2] dedicated documentation page for detailed options.
-    
+### Define the search query
 
-## Notifications and log samples
+As you define the search query, the graph above the search fields updates.
 
-It is possible to add up to 10 samples of logs that triggered the monitor in the notification message.
-This is available for Slack, Jira, Webhook, Microsoft Teams, and email notifications.
+* If you have [multiple log indexes][2], select the index to search.
+* Construct a search query using the same logic as a [log explorer search][3].
+* Choose a [facet][4] or [measure][5] to monitor.
+    * Facets display `Count`.
+    * Measures display `Measure` with an aggregation choice: `min`, `avg` (default), `sum`, `median`, `pc75`, `pc90`, `pc95`, `pc98`, `pc99`, or `max`.
+* Define the alert grouping (optional). **Note**: With or without alert grouping defined, you get **one** alert when the aggregated value meets the set conditions. Even if you split the query by host, a single notification is sent if several hosts meet the set conditions. This is done to reduce notification noise.
 
-* Samples are not displayed for recovery notifications.
+### Set alert conditions
 
- **Enabling log samples in notifications**:
-    
-  {{< img src="monitors/monitor_types/log/activate-log-monitor-sample.png" alt="Activate log samples in message" responsive="true" style="width:50%;" >}}
-    
-  **Example for a Slack notification** 
+* Trigger when the metric is `above`, `above or equal to`, `below`, or `below or equal to`
+* the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc.
+* Alert threshold `<NUMBER>`
+* Warning threshold `<NUMBER>`
 
-  {{< img src="monitors/monitor_types/log/slack-log-sample.png" alt="Slack notification example" responsive="true" style="width:50%;" >}}
- 
-### Notifications for groups
+#### No data and below alerts
 
-Notifications from monitors split by group may include the list of the top 10 of breaching values instead of 10 log samples.
+To receive a notification if a specific set of logs are not received anymore, set the condition `below 1`. This notifies when no logs match the monitor query in the given timeframe. 
```

Comment:
This concept is really confusing to people, especially with group-by behavior. We should be even more explicit here. Perhaps:
>To receive a notification when all groups in a service have stopped sending logs, set the condition to `below 1` This notifies when no logs match the monitor query in a given timeframe across all aggregate groups.